app/scripts/controllers/permissions/methodMiddleware.js

```diff
@@ -42,6 +38,7 @@ export default function createMethodMiddleware ({
 
         // get the accounts again
         accounts = await getAccounts()
+        /* istanbul ignore else: too hard to induce */
```

Comment:
Hmm

Reply:
Yeah, the `else` case should _always_ be handled by the `catch` clause on ~L34 above. This is the equivalent of a catastrophic error that I can't anticipate how it would actually happen in practice, but want to cover for in case it does.

I could induce it by means of some of the Promise hackery I've done to make these tests, but I don't think it's worth it.

Reply:
It could happen in practice if the permission was granted and then revoked in quick succession. Seems unlikely but possible

Reply:
Yeah exactly, in my mind a strange combination of a dapp requesting permissions and a user accepting and removing them almost simultaneously.

Reply:
This is the one `istanbul ignore` I left in, btw.